backend/api/Areas/Leases/Mapping/Lease/InsuranceMap.cs

```diff
@@ -23,7 +23,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.StartDate, src => src.StartDate)
                 .Map(dest => dest.ExpiryDate, src => src.ExpiryDate)
                 .Map(dest => dest.RiskAssessmentCompletedDate, src => src.RiskAssessmentCompletedDate)
-                .Map(dest => dest.InsuranceInPlace, src => (src.StartDate > System.DateTime.Now) && (System.DateTime.Now < src.ExpiryDate));
+                .Map(dest => dest.InsuranceInPlace, src => (src.StartDate.Date >= System.DateTime.Now.Date) && (System.DateTime.Now.Date < src.ExpiryDate.Date));
```

Comment:
I could be wrong, but my understanding of expiry dates is that they are inclusive, did you check this with the business?